Documentation/Books/AQL/Graphs/KShortestPaths.md

```diff
@@ -0,0 +1,174 @@
+k Shortest Paths in AQL
+====================
+
+General query idea
+------------------
+
+This type of query is supposed to find the first *k* paths in order of length
+(or weight) between two given documents, *startVertex* and *targetVertex* in
+your graph.
+
+Every such path will be returned as a JSON object with three components:
+
+1. an array of the vertices on the path,
+2. an array of the edges on the path, and
+3. the weight of the path.
```

Comment:
Are the attributes called `vertices`, `edges` and `weight`? Then we could hint this here already:

1. an array of the `vertices` on the path,
2. an array of the `edges` on the path, and
3. the `weight` of the path.

```
1. an array of the `vertices` on the path,
2. an array of the `edges` on the path, and
3. the `weight` of the path.
```